packages/postgres-database/src/simcore_postgres_database/migration/versions/da8abd0d8e42_add_comp_runs_table.py

```diff
@@ -31,7 +32,7 @@ def upgrade():
         sa.Column("iteration", sa.BigInteger(), autoincrement=False, nullable=False),
         sa.Column(
             "result",
-            sa.dialects.postgresql.ENUM(
+            ENUM(
```

Comment:
I am not so sure whether it is good idea to modify migration script. I would propose to ignore this errors by adding this to `mypy.ini`

[mypy-simcore_postgres_database.migration.versions.*]
ignore_errors = true